include/benchmark/benchmark_api.h

```diff
@@ -858,6 +861,10 @@ class Fixture : public internal::Benchmark {
 #define BENCHMARK_MAIN()                   \
   int main(int argc, char** argv) {        \
     ::benchmark::Initialize(&argc, argv);  \
+    if (argc > 1) {                        \
```

Comment:
Why have this check at all? `ReportUnrecognizedArguments` handles that case already, doesn't it?